package.json

```diff
@@ -20,7 +20,8 @@
     "express": "^4.17.1",
     "joi": "^17.4.2",
     "jsonwebtoken": "^8.5.1",
-    "pg": "^8.7.1"
+    "pg": "^8.7.1",
+    "react-transition-group": "^4.4.2"
```

Comment:
why you installed this here ?

Reply:
It's not installing, after pulling the main branch  and the ( "react-transition-group": "^4.4.2") module added there is a ( , ) added in the package Json file